actions/skipafter.go

```diff
@@ -23,7 +23,6 @@ func (a *skipafterFn) Init(_ rules.RuleMetadata, data string) error {
 }
 
 func (a *skipafterFn) Evaluate(r rules.RuleMetadata, tx rules.TransactionState) {
-	tx.DebugLogger().Debug("[%s] Starting secmarker %q", tx.ID(), a.data)
```

Comment:
@jptosso is this correct?

Reply:
Coincidentally, I'm debugging SkipAfter actions. At debug level, this message seems reasonable to me:
```
[DEBUG] [fahtwIdZavCjGThemMU] [980041] Evaluating action skipAfter for rule 980041
[DEBUG] [fahtwIdZavCjGThemMU] Starting secmarker "END-REPORTING"
[DEBUG] [fahtwIdZavCjGThemMU] rule 980041 matched
[DEBUG] [fahtwIdZavCjGThemMU] [980041] Finish evaluating rule 980041
[DEBUG] [fahtwIdZavCjGThemMU] Skipping rule 980170 because of SkipAfter, expecting END-REPORTING and got: ""
```

Reply:
we can change the message in future PRs, but it's a useful debug message